### PR TITLE
Update Windows Requirement

### PR DESCRIPTION
**Pull request by ahmaddanialmohd**
#### Summary
Remove Windows 7+ from **line 32** since it has already reached its EOL.

### CODE REVIEW

**Review by amyblais — CHANGES_REQUESTED** [review]
Does this need to be announced to customers?

**Comment by cwarnermm** [comment]
I imagine that there are additional references to Windows 7 that should also be removed for consistency. I've logged https://mattermost.atlassian.net/browse/MM-35873 for this effort.

File: source/install/requirements.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Desktop Apps
 .. csv-table::
     :header: "Operating System", "Technical Requirement"
 
-    "Windows", "Windows 7, 8.1, and 10"
+    "Windows", "Windows 8.1, and 10"
```
Comment (cwarnermm) [inline source/install/requirements.rst:32]:
```suggestion
    "Windows", "Windows 8.1 and 10"
```

**Review by cwarnermm — APPROVED** [review]
Approved with the change proposed inline

**Review by esethna — APPROVED** [review]

**Comment by cwarnermm** [comment]
@esethna - See Amy's question above about whether or not we should announce that we've formally dropped support for Windows 7.

**Comment by esethna** [comment]
@amyblais @cwarnermm yes we should probably do some communication around this. @amyblais I'll leave this in your court to figure out. 0/5, but perhaps we do this with the v6.0 launch among other deprecations/depreciations to make the communication easier. 

Some resources that may help with the communication: https://support.microsoft.com/en-us/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962

**Comment by cwarnermm** [comment]
@amyblais - Is additional work remaining on this ticket?

**Comment by amyblais** [comment]
@cwarnermm No, this can be merged with v6.0 docs, or a new PR can be submitted since this has conflicts.

**Comment by cwarnermm** [comment]
Thanks, @amyblais. I agree that trying to manage the conflicts in this PR will take far more time than applying this update in the ``v6.0-documentation`` branch. I'll get this update in today.

**Comment by cwarnermm** [comment]
Closing this ticket since the changes required have been applied via https://github.com/mattermost/docs/pull/5110 for v6.0.

**Pull request closed without merging**